dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/DataSourceService.java

```diff
@@ -607,7 +607,10 @@ protected void init() throws Exception {
             } else if (Driver.class.getName().equals(type)) {
                 ifc = Driver.class;
                 String url = vProps.getProperty("URL", vProps.getProperty("url"));
-                vendorImpl = jdbcDriverSvc.getDriver(url, vProps, id);
+                if (url != null && !"".equals(url)) {
```

Comment:
There are two places that call getDriver - init() which you updates here and getManagedConnectionFactory().  I assume getManagedConnectionFactory should be updated with the null check as well?